src/dojos/cd-dojo-details.vue

```diff
@@ -106,7 +106,17 @@
     },
     computed: {
       address() {
-        return !this.dojoDetails.address1 ? undefined : [
+        if (!this.dojoDetails.address1) {
+          return undefined;
+        } else if (this.dojoDetails.address1.includes(this.dojoDetails.countryName)) {
+          return this.dojoDetails.address1;
+        } else if (this.dojoDetails.address1.includes(this.dojoDetails.placeName)) {
```

Comment:
placeName is the address returned by GAPI. i wouldn't use it as a reference as it may totally differ